client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.js

```diff
@@ -92,7 +92,7 @@ function WPLineItem( {
 			data-product-type={ item.type }
 		>
 			<LineItemTitle id={ itemSpanId } isSummary={ isSummary }>
-				{ item.label }
+				{ item.label } { isTitanMail && <>(Monthly)</> }
```

Comment:
Shouldn't this string be localized? Can't we actually include this string in the item label? We sell other products on WordPress.com that are on a monthly billing, how do we should that information to users in these cases?

Reply:
This might be what you're looking for: https://github.com/Automattic/wp-calypso/blob/93cb8c82e1edd4bbc179af50e5b6e082b361e3ca/client/my-sites/checkout/composite-checkout/lib/translate-cart.ts#L426-L435

That code sets the `sublabel` property of the `WPCOMCartItem` object, which is rendered a few lines down from here by `LineItemSublabelAndPrice`.

Reply:
Thanks for the nudge - it totally skipped my mind i.e. localisation. @sirbrillig - thanks for setting me straight, but actually the issue is because that piece of code is restricted to plans etc